Source/cmake/GStreamerDefinitions.cmake

```
@@ -2,6 +2,7 @@ WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_VIDEO PUBLIC ON)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_WEB_AUDIO PUBLIC ON)
 
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_MEDIA_SOURCE PUBLIC ON)
+WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_MANAGED_MEDIA_SOURCE PUBLIC ON)
```

The new option needs to be PRIVATE, and might as well drive-by change the existing option too, because there's no longer any justifiable reason for it to be PUBLIC:

```suggestion
WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_MEDIA_SOURCE PRIVATE ON)
WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_MANAGED_MEDIA_SOURCE PRIVATE ON)
```